CodeConverter/Util/FromRoslyn/VbConversionKind.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace ICSharpCode.CodeConverter.Util.FromRoslyn;
+
+[Flags]
+internal enum VbConversionKind
+{
+    // If there is a conversion, either [Widening] or [Narrowing] bit must be set, but not both.
+    // All VB conversions are either Widening or Narrowing.
+    // 
+    // To indicate the fact that no conversion exists:
+    // 1) Neither [Widening] nor [Narrowing] are set.
+    // 2) Additional flags may be set in order to provide specific reason.
+    // 
+    // Bits from the following values are never set at the same time :
+    // Identity, Numeric, Nullable, Reference, Array, TypeParameter, Value, [String], WideningNothingLiteral, InterpolatedString
+
+    FailedDueToNumericOverflow = 1 << 31, // Failure flag
+    FailedDueToIntegerOverflow = FailedDueToNumericOverflow | 1 << 30, // Failure flag
+    FailedDueToNumericOverflowMask = FailedDueToNumericOverflow | FailedDueToIntegerOverflow,
```

Comment:
Probably worth putting a doc comment on the enum for which namespace it's originally from. I [couldn't find it here](https://github.com/dotnet/roslyn/search?q=VbConversionKind)
Assuming it's a verbatim copy, let's suppress the compiler warning once at the top of the file.

Reply:
Ah, found it: https://github.com/dotnet/roslyn/blob/main/src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb#L282
Since it's a conversion of that code, it's probably worth putting a link in directly, then either removing the duplicate member or suppressing the error

Reply:
Added a permalink and suppressed the warning.
I have to enable treat warnings as errors. I'm used to having them red not orange.